spectrum/test_article.py

```diff
@@ -47,6 +47,9 @@ def test_article_multiple_versions(generate_article, modify_article):
     version1_content = checks.JOURNAL.article(id=article.id(), volume=article_from_api['volume'], version=1)
     assert 'cytomegalovirus' in version1_content
     assert 'CYTOMEGALOVIRUS' not in version1_content
+    version2_content_cdn = checks.JOURNAL_CDN.article(id=article.id(), volume=article_from_api['volume'], version=2)
```

Comment:
The test is about publishing two versions, so we are checking that the v1 on journal doesn't change and that the v2 appears on the CDN. Testing that v1 on cdn doesn't change would not be of much value because it could just be a cached result.